felix/dataplane/linux/int_dataplane.go

```diff
@@ -750,13 +753,31 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 			log.Info("BPF enabled but no Kubernetes client available, unable to run kube-proxy module.")
 		}
 
-		if config.BPFConnTimeLBEnabled {
+		if config.BPFConnTimeLB == string(apiv3.BPFConnectTimeLBDisabled) &&
+			config.BPFHostNetworkedNAT == string(apiv3.BPFHostNetworkedNATDisabled) {
+			log.Warn("Access to services from host networked process wont work, forcing hostnetworked NAT to Enabled")
+			config.BPFHostNetworkedNAT = string(apiv3.BPFHostNetworkedNATEnabled)
+		}
+		// HostNetworkedNAT is Enabled and CTLB enabled.
+		// HostNetworkedNAT is Disabled and CTLB is either disabled/TCP.
+		// The above cases are invalid configuration. Revert to CTLB enabled.
+		if config.BPFHostNetworkedNAT == string(apiv3.BPFHostNetworkedNATEnabled) {
+			if config.BPFConnTimeLB == string(apiv3.BPFConnectTimeLBEnabled) {
+				log.Warn("Access to services may not work properly, reverting to default CTLB configuration")
```

Comment:
`config.BPFHostNetworkedNAT == string(apiv3.BPFHostNetworkedNATEnabled` doesnot make sense, but also would not do much harm. I would revert here to Disabled instead of flipping the other to TCP.

Reply:
I think flipping the ```ConnTimeLB``` to ```TCP``` keeps it line with the defaults.